### DISCUSSION TITLE
Composer install - needing authentication for drupal/coder 8.3.13 of git.drupalcode.org

### DISCUSSION THREAD

**Discussion by eieiaungtpicap** [composer/composer#10181] [Q&A / Support]
Composer install - needing authentication for drupal/coder 8.3.13 of git.drupalcode.org
Hi Everyone, 
I'm very new to Composer and trying to setup existing drupal project to my local using composer. 
I have all ssh key setup and installed php, configure php.ini as well. 
But not sure why I'm getting to key in the username and password for the authentication of drupalcode.org whenever install composer. 
Any advices or anything I missed? 

Thanks in advance. 

![image](https://user-images.githubusercontent.com/92393023/137699633-da3555c1-2100-4881-a976-fc3991474779.png)

**Comment by Seldaek** [comment]
I'm not sure either tbh.. Seems cloning the repo does not require authentication. Could be some environment issue, or something that escapes me right now.

**Reply by Nguimjeu** [reply]
I have the same issue with _drupal/coder_. No idea how to solve that.

**Comment by alainmaradan** [comment]
Deleting the composer.lock solve the issue for me

**Reply by jmsantos01** [reply]
Delete composer.lock in the drupal folder

**Reply by devkinetic** [reply]
This is not a solution. The lock file maintains the current versions of all the packages.
